examples/eventlisteners/eventlistener-triggerref.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: triggers.tekton.dev/v1alpha1
+kind: EventListener
+metadata:
+  name: listener-triggerref
+spec:
+  serviceAccountName: tekton-triggers-example-sa
+  triggers:
+    - triggerRef: trigger
```

Comment:
Can we move the Trigger type to this file as well. Would make the example easier to understand.

Reply:
Done